.github/workflows/generate_and_push_deriv_api_types.yml

```diff
@@ -77,24 +77,22 @@ jobs:
 
             # lerna: To allow us to bootstrap the project.
             # json: To manually edit the target package's package.json
-            npm i -g lerna@latest json@latest
+            npm i -g lerna@latest json@latest npm@8
```

Comment:
changes to this file is not related to POO, right?

Reply:
Hello @ali-hosseini-fs, 

This change is from upstream. It's @balakrishna-binary's change not mine. I just pulled it in

Reply:
He does it in this [commit](https://github.com/binary-com/deriv-app/pull/5886/commits/1dd5595f6a4605d7c62609bce5e9ec0d62f578f6):